StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentMethodsView.swift

```diff
@@ -381,7 +381,8 @@ class EmbeddedPaymentMethodsView: UIView {
                 self.layoutIfNeeded()
             }
         case (true, false): // Hidden -> Hidden
-            break
+            // Update mandate text on the view so the payment option we vend to the merchant has the correct mandate
+            self.mandateView.attributedText = mandateText
```

Comment:
The payment option that embedded vends is a computed property and looks at mandateView.attributedText to vend the mandate. So even if we are not showing the mandate ourselves, we should at least update the string on the view so we can vend a mandate to the merchant.